core/src/modules/ops/destination.ts

```diff
@@ -912,13 +913,41 @@ export namespace DestinationOps {
     success: boolean;
     error: Error;
   }> {
-    const _exports: Export[] = []; // only ones we are sending
+    const _exports: Export[] = [];
+
     for (const _export of givenExports) {
-      // TODO: maybe should recalcuate hasChanges and from current mostRecent
-      if (!_export.hasChanges) {
-        await _export.complete(); // do not include exports with hasChanges=false
+      const { isLocked, lockedBy } = await getLock(
+        `${_export.recordId}:${_export.destinationId}`,
```

Comment:
👍 the lock key is a tuple comprised of the record.id and destination.id

Reply:
I think `lockedBy` is a random UUID, so won't the `await Export.findById(lockedBy);` below always fail to find an Export?  I'll add an idea below to not need store any information in the lock